Packaging/nix/org.diasurgical.DevilutionX.yml

```diff
@@ -4,13 +4,16 @@ runtime-version: '21.08'
 sdk: org.freedesktop.Sdk
 command: devilutionx
 rename-desktop-file: devilutionx.desktop
+rename-appdata-file: devilutionx.metainfo.xml
+rename-icon: devilutionx
+
 finish-args:
     - --socket=fallback-x11
     - --socket=wayland
     - --device=dri
     - --share=network
+    - --share-ipc
     - --socket=pulseaudio
-    - --filesystem=xdg-config
```

Comment:
Is this not needed?

Reply:
We already have write access in `~/.var/app/$FLATPAK_ID`
Changing the path of diablo.ini from `~/.var/app/$FLATPAK_ID/data/diasurgical/devilution`, which is `XDG_DATA_HOME` to `~/.var/app/$FLATPAK_ID/config`, which is `XDG_CONFIG_HOME` does not require additional permissions